src/components/CourseTable/index.jsx

```diff
@@ -44,6 +44,7 @@ class CourseTable extends React.Component {
             { value: PUBLISHED, label: 'Published', color: '#008100' },
             { value: REVIEWED, label: 'Scheduled', color: '#0075b4' },
             { value: 'unsubmitted', label: 'Unsubmitted', color: '#E2C018' },
+            { value: ARCHIVED, label: 'Archived', color: '#e7e7e7' },
```

Comment:
Please use a different color for differentiation.

Reply:
Anyways, I changed it to badge-secondary (#454545) color.